unitylibs/core/workflow/workflow-acrobat/service-handler.js

```diff
@@ -27,13 +27,21 @@ export default class ServiceHandler {
   async fetchFromService(url, options) {
     const response = await fetch(url, options);
     const error = new Error();
+    const contentLength = response.headers.get('Content-Length');
     if (response.status !== 200) {
+      if (contentLength !== '0') {
+        const resJson = await response.json();
+        if (resJson.reason?.includes('quotaexceeded')) {
```

Comment:
is there an actual use-case where the reason would include both `quotaexceeded` and `notentitled`? if not, I think we can rewrite as
```js
['quotaexceeded', 'notentitled'].forEach((errorMessage) => {
  if (resJson.reason?.includes(errorMessage)) error.message = errorMessage;
});
```

Reply:
No, there shouldn't be an actual use-case where both would be included. Have updated this accordingly.